src/legacy/core_plugins/kibana/public/dashboard/dashboard_app.js

```diff
@@ -231,8 +231,6 @@ app.directive('dashboardApp', function ($injector) {
         $scope.indexPatterns = dashboardStateManager.getPanelIndexPatterns();
       };
 
-      $scope.$watch('model.query', $scope.updateQueryAndFetch);
```

Comment:
Removing this line does prevent the duplicate network call, but it also makes it so that if you change the query in the URL (and not in the query bar), then saved searches aren't updated.

Reply:
Cool, added a test that would have caught that and confirmed it fails:
<img width="1162" alt="screen shot 2018-12-18 at 2 43 59 pm" src="https://user-images.githubusercontent.com/16563603/50178780-6f4a3500-02d3-11e9-84a6-84b120e78832.png">